package.json

```diff
@@ -11,58 +11,58 @@
     "node": ">=10.13"
   },
   "scripts": {
-    "build-all": "npm-run-posix-or-windows build-all:task",
-    "build-all:task": "yarn build-extension & yarn build-devtools & yarn build-lr & yarn build-viewer & wait",
     "build-all:task:windows": "yarn build-extension && yarn build-devtools && yarn build-lr && yarn build-viewer",
-    "build-extension": "node ./build/build-extension.js",
+    "build-all:task": "yarn build-extension & yarn build-devtools & yarn build-lr & yarn build-viewer & wait",
+    "build-all": "npm-run-posix-or-windows build-all:task",
     "build-devtools": "node ./build/build-bundle.js clients/devtools-entry.js dist/lighthouse-dt-bundle.js && node ./build/build-dt-report-resources.js",
+    "build-extension": "node ./build/build-extension.js",
     "build-lr": "node ./build/build-lightrider-bundles.js",
+    "build-proto-roundtrip": "cd proto/scripts && PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python json_roundtrip_via_proto.py",
     "build-viewer": "node ./build/build-viewer.js",
+    "bundlesize": "bundlesize",
+    "changelog": "conventional-changelog --config ./build/changelog-generator/index.js --infile changelog.md --same-file",
+    "chrome": "node lighthouse-core/scripts/manual-chrome-launcher.js",
     "clean": "rimraf dist proto/scripts/*.json proto/scripts/*_pb2.* proto/scripts/*_pb.* proto/scripts/__pycache__ proto/scripts/*.pyc *.report.html *.report.dom.html *.report.json *.devtoolslog.json *.trace.json lighthouse-core/lib/i18n/locales/*.ctc.json || true",
+    "cli-unit": "yarn unit-cli",
+    "codecov": "codecov -f unit-coverage.lcov -F unit && codecov -f smoke-coverage.lcov -F smoke",
+    "compile-devtools": "bash lighthouse-core/scripts/compile-against-devtools.sh",
+    "compile-proto": "protoc --python_out=./ ./proto/lighthouse-result.proto && mv ./proto/*_pb2.py ./proto/scripts || (echo \"❌ Install protobuf ≥ 3.7.1 to compile the proto file.\" && false)",
+    "core-unit": "yarn unit-core",
```

Comment:
I don't think it improves things to move all the `*-unit` aliases sprinkled in alphabetical instead of colocated

![image](https://user-images.githubusercontent.com/2301202/61881495-4ff43580-aebc-11e9-9302-20a279a88e7b.png)

Reply:
we can rename these too. i  mean comeon:

```
"unit-viewer": "jest \"lighthouse-viewer/.*-test.js\""
...
"viewer-unit": "yarn unit-viewer",
```

:P

```
unit:viewer
unit:core
unit:cli
```
?